### ISSUE TITLE
try.nodejs.org

### ISSUE THREAD

**Issue by hemanth** [nodejs/nodejs.org-archive#10]
I'm aware that there are many such web based REPLs out there, but would it not be fun to have a http://try.nodejs.org :question: 

**Update** //CC @isaacs

**Comment by matthewlopez0977** [comment]
I agree. It'll allow poeple to experience node before downloading. I'll like to add to this proposal, that not only this be carry out, but let it be so with: http://nodeschool.io

Online virtual nodes to learn experience would eases many JavaScript developer to trying out nodes. 

When you are console base, it can be intimidating. Having an online environment like walk through nodes, will help javascript developer ease  into it.

**Comment by ThisIsMissEm** [comment]
Might be a way to do this with Docker and a machine, or smartos zones.
